lib/console.c

```diff
@@ -80,7 +81,7 @@ console_is_present(void)
 
 /* re-acquire a console after startup using an array of fds */
 gboolean
-console_acquire_from_fds(gint fds[3])
+console_acquire_from_fds(gint fds[3], gboolean backup_initial_console)
```

Comment:
this argument is always TRUE, so I don't think we need it. It should just always save the fds instead.

Reply:
it was a bug, i forgot to update with the remote param `cmd_args.restore_console_on_release`, fixed